prime-angular-frontend/src/app/lib/modules/ngx-busy/busy.service.ts

```diff
@@ -18,14 +18,21 @@ export class BusyService {
   /**
    * @description
    * @param message the message to be displayed when a busy has a busy thing (subscription, promise, array of either... etc)
-   * @param request the observable that will run while the busy message and spinner are shown
+   * @param observableOrFunction the observable or function that will run while the busy message and spinner are shown
```

Comment:
The name of the function and name/typing of the params explains their function.  A short description would be sufficient, and more likely to be read:

```
@description
Pipe for setting/clearing the busy message that wraps the execution of a customizeable operator function.
```